pkg/ui/query.go

```diff
@@ -52,7 +53,9 @@ func NewQueryUI(logger log.Logger, flagsMap map[string]string) *Query {
 
 func (q *Query) Register(r *route.Router) {
 	r.Get("/", func(w http.ResponseWriter, r *http.Request) {
-		http.Redirect(w, r, "/graph", http.StatusFound)
+		// see description of PathPrefixStrip at https://docs.traefik.io/basics/#frontends
```

Comment:
Not correct comment format (should be full sentence - starting with upper case first letter)